app/models/channel.rb

```diff
@@ -112,6 +113,17 @@ class Channel < ApplicationRecord
     where.not(contested_by_channel_id: nil).where("contest_timesout_at < ?", Time.now)
   }
 
+  scope :missing_deposit_id, -> { where(deposit_id: nil) }
+  scope :using_active_bitflyer_connection, -> {
+    joins(:publisher)
```

Comment:
this might actually be redundant but the specs work.  I could remove it later

Reply:
This would be useful to have for Uphold and Gemini too, eventually.

Reply:
@tsmartt For sure, pattern is the same.

Reply:
maybe just `with_active_bitflyer_connections`

Reply:
Yeah, some standard would be better.  Not overthinking it atm.  If/when we add the others we can clean it up.